packages/adblocker/test/parsing.test.ts

```diff
@@ -2444,4 +2444,12 @@ describe('scriptlets arguments parsing', () => {
       );
     }
   });
+
+  it('parses replace modifier', () => {
```

Comment:
It would be nice to add more tests since anything related to HTML filtering is critical to get right. If possible, let's add one test case for each existing filter in the lists used so far (I assume there are not so many). Let's particularly try to find corner cases of the behavior of this new `$replace` option.

We also need more end-to-end tests on the whole HTML filtering.